HelloBlockAsync/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "PluginName": "Hello Async",
+    "PluginDescription": "Testing async JS API",
+    "PluginType": "Panel",
+    "Scripts": [
+        "PLUGINLOCATION/empty.js"
+    ],
```

Comment:
I'd like to get rid of this for this example, since it is not used. I think that would require an additional change in FormIt.

Reply:
Does FormIt choke if you remove "Scripts" prop?

Reply:
I don't what happens exactly, but the plugin just doesn't show up. I need to step through on the FormIt side.